spec/javascripts/govuk_publishing_components/analytics-ga4/ga4-link-tracker.spec.js

```diff
@@ -143,6 +127,7 @@ describe('GOVUK.analyticsGa4.linkTracker', function () {
 
       GOVUK.triggerEvent(nestedImage, 'click')
 
+      expected.event_data.link_domain = 'http://www.nationalarchives.gov.uk'
```

Comment:
Any reason why sometimes the expected `link_domain` is hardcoded and sometimes it reads it from the attribute? Might be good to be consistent if possible?

Reply:
@andysellick Some tests always have the same domain name, so I could hard code it in the test whereas some tests had a bunch of different domains, so I had to mark them up so the `expected` value would change depending on the domain. Should I use the `link_domain` attribute everywhere instead?

Reply:
No, that makes sense, thanks.